src/Application/RectorApplication.php

```diff
@@ -122,7 +122,12 @@ public function runOnFileInfos(array $phpFileInfos): void
 
         if (! $this->symfonyStyle->isVerbose() && $this->configuration->showProgressBar()) {
             // why 5? one for each cycle, so user sees some activity all the time
-            $this->symfonyStyle->progressStart($fileCount * 5);
+            $stepMultiplier = 4;
+            if ($this->fileSystemFileProcessor->getFileSystemRectorsCount() !== 0) {
+                ++$stepMultiplier;
+            }
+
+            $this->symfonyStyle->progressStart($fileCount * $stepMultiplier);
 
```

Comment:
@dobryy This is the fix for

![image](https://user-images.githubusercontent.com/924196/88484828-fa8cb180-cf71-11ea-8d6f-6a4f723b1dcb.png)